### PR TITLE
fix module object has no attribute NodeDef for tensorflow 0.11

### PR DESCRIPTION
**Pull request by qingzew**
reference https://github.com/tensorflow/tensorflow/issues/4759 and https://github.com/tensorflow/models/pull/520

### CODE REVIEW

File: inception/inception/slim/variables.py [hunk 1]
```diff
@@ -84,7 +84,7 @@
 
 import tensorflow as tf
 
-from tensorflow.core.framework import graph_pb2
```
Comment (drpngx) [inline inception/inception/slim/variables.py]: Remove both these lines

File: inception/inception/slim/variables.py [hunk 2]
```diff
@@ -211,7 +211,7 @@ def variable_device(device, name):
   """Fix the variable device to colocate its ops."""
   if callable(device):
     var_name = tf.get_variable_scope().name + '/' + name
-    var_def = graph_pb2.NodeDef(name=var_name, op='Variable')
```
Comment (drpngx) [inline inception/inception/slim/variables.py]: Use the blessed way: tf.NodeDef

**Comment by qingzew** [comment]
@drpngx I have change it

**Comment by drpngx** [comment]
Thanks!

**Pull request merged**